cylc/flow/cfgspec/workflow.py

```diff
@@ -694,8 +695,21 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     Assigned tasks will automatically be removed
                     from the default queue.
                 ''')
-            with Conf('default', meta=Queue):
-                Conf('limit', VDR.V_INTEGER, 0)
+            with Conf('default', meta=Queue, desc='''
+                The default queue - all tasks are assigned to this queue.
+                Designed to allow you to set
+                :cylc:conf:`flow.cylc[scheduling][queues][default]limit`.
```

Comment:
```suggestion
                The default queue - all tasks not assigned to other queues are assigned to this one.
```
Not really necessary to mention the limit, which is the very next item anyway.